client/src/commons/TypePreventiveTextField/TypePreventiveTextField.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { TextField, Tooltip } from '@material-ui/core';
+
+import get from 'Utils/auxiliaryFunctions/auxiliaryFunctions'
+
+import TypePreventiveTextFieldType from './TypePreventiveTextFieldTypes';
+
+const TypePreventiveTextField: TypePreventiveTextFieldType = (props) => {
+  const value = (props.value == null || props.value === undefined) ? "" : props.value;
```

Comment:
why not ===

Reply:
changed to !props.value ? "" : props.value;